system/jlib/jset.cpp

```diff
@@ -476,6 +480,17 @@ class CBitSet : public CBitSetBase<CBitSetMemoryHelper>
             memset(mem, 0, bitSetUnits*sizeof(bits_t));
         fixedMemory = true;
     }
+    CBitSet(unsigned maxBits, bool reset)
```

Comment:
The code used by createBitset dynamically increases the size of the memory buffer available - there isn't any advantage in preallocating it, and I don't see any advantage to restricting it to a particular size.